README.md

```diff
@@ -1 +1,57 @@
-# vba-migrator-addin
+# Project Title
+### vba-migrator-addin
+VBA Editor add-in to show compatibility with Google Sheets.
+
+## About
+For now this Add-in will give you a button in your VBA editor.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+```
+Having Window OS
+```
+
+## Deployment
+
+Here is the registry .key script to register the Addin, note you will need to change some of the settings in order to register it properly. So open your favourite text editor copy-paste the following text in it and save it as .reg file, don't forget to change the path of the your .dll file here "CodeBase"="file:///PathToAssembly". One more important thing is if your VBA editor is 32 bit then the registry should be made in Addins so if it is the case change Addins64 ~ Addins.  
```

Comment:
This paragraph is not clear. Please fix.

Reply:
Tried please take a look.